Dashed namespace crashes exporter

**Issue type**: Bug report  

**What happened**:

I added a (yaml) configuration and used a namespace name containing a dash - in the same way as can be seen in the README, but Prometheus seems to crash since a dash is not valid in a metric name.

**What you expected to happen**:

In my case I would like it the most if dashes are converted to underscores since that would save be word in pre-formatting the string in Ansible. But for the general use case that might be too much magic. 
I think it is not a problem that the exporter crashes since this is a limitation coming from Prometheus itself, but at least examples (such in the README) should be updated accordingly to represent actually working code.

**How to reproduce it (as minimally and precisely as possible)**:

Simply start with any config that has a dash in the namespace name.

**Other important information**:

Thank you for creating this awesome exporter! It gives us super useful information and is in general super simple to use!

**Configuration file** (remove section, if not applicable):

```yml
listen:
  port: 4040
  address: "127.0.0.1"
namespaces:
- name: xxx-dev
  format: '$host - $remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent"
"$http_x_forwarded_for" rt="$request_time" uct="$upstream_connect_time" uht="$upstream_header_time" urt="$upstream_response_time"'
  source_files:
    - './access.log'
```

**Exporter output** (remove section, if not applicable):

```
loading configuration file ./nginx_exporter.yml
using configuration {{%!s(int=4040) 127.0.0.1} {%!s(bool=false)     {  []}} [{xxx-dev [./access.log] $host - $remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for" rt="$request_time" u
ct="$upstream_connect_time" uht="$upstream_header_time" urt="$upstream_response_time" map[] [] [] []}] %!s(bool=false)}
starting listener for namespace xxx-dev
running HTTP server on address 127.0.0.1:4040
panic: descriptor Desc{fqName: "xxx-dev_http_response_count_total", help: "Amount of processed HTTP requests", constLabels: {}, variableLabels: [method status]} is invalid: "xxx-dev_http_response_count_total" is not a valid
 metric name

goroutine 21 [running]:
github.com/martin-helmich/prometheus-nginxlog-exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc4200a2440, 0xc420176160, 0x1, 0x1)
        /home/travis/gopath/src/github.com/martin-helmich/prometheus-nginxlog-exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:404 +0x9e
github.com/martin-helmich/prometheus-nginxlog-exporter/vendor/github.com/prometheus/client_golang/prometheus.MustRegister(0xc420176160, 0x1, 0x1)
        /home/travis/gopath/src/github.com/martin-helmich/prometheus-nginxlog-exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:152 +0x53
main.(*Metrics).Init(0xc42017c100, 0xc42006ff58)
        /home/travis/gopath/src/github.com/martin-helmich/prometheus-nginxlog-exporter/main.go:103 +0x9c2
main.main.func2(0xc42009ab50, 0x9, 0xc4200892c0, 0x1, 0x1, 0xc42010a500, 0xfa, 0xc420130060, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/martin-helmich/prometheus-nginxlog-exporter/main.go:177 +0x8d
created by main.main
        /home/travis/gopath/src/github.com/martin-helmich/prometheus-nginxlog-exporter/main.go:173 +0x5db
```

**Environment**:
- Exporter version: 1.2.0

--- a/README.adoc
+++ b/README.adoc
@@ -88,7 +88,7 @@ consul {
   }
 }
 
-namespace "app-1" {
+namespace "app1" {
   format = "$remote_addr - $remote_user [$time_local] \"$request\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\" \"$http_x_forwarded_for\""
   source_files = [
     "/var/log/nginx/app1/access.log"
@@ -100,7 +100,7 @@ namespace "app-1" {
   }
 }
 
-namespace "app-2" {
+namespace "app2" {
   format = "$remote_addr - $remote_user [$time_local] \"$request\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\" \"$http_x_forwarded_for\" $upstream_response_time"
   source_files = [
     "/var/log/nginx/app2/access.log"
@@ -128,7 +128,7 @@ consul:
     tags: ["foo", "bar"]
 
 namespaces:
-  - name: app-1
+  - name: app1
     format: "$remote_addr - $remote_user [$time_local] \"$request\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\" \"$http_x_forwarded_for\""
     source_files:
       - /var/log/nginx/app1/access.log
@@ -136,7 +136,7 @@ namespaces:
       app: "application-one"
       environment: "production"
       foo: "bar"
-  - name: app-2
+  - name: app2
     format: "$remote_addr - $remote_user [$time_local] \"$request\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\" \"$http_x_forwarded_for\" $upstream_response_time"
     source_files:
       - /var/log/nginx/app2/access.log
@@ -177,7 +177,7 @@ the experimental feature originally introduced in #23):
 
 [source,hcl]
 ----
-namespace "app-1" {
+namespace "app1" {
   // ...
 
   relabel "request_uri" {

